src/code/entities/07_master_stalfos.asm

```diff
@@ -424,7 +424,7 @@ func_007_6BC8::
     call IncrementEntityState                     ; $6BDC: $CD $12 $3B
 
 label_007_6BDF:
-    call EntityLinkPositionXDifference_07         ; $6BDF: $CD $5D $7E
+    call GetEntityXDistanceToLink_07              ; $6BDF: $CD $5D $7E
```

Comment:
Thanks again for restoring some sanity and consistency into the naming of all these different helpers 🙏